lib/vquic/ngtcp2.c

```diff
@@ -1830,6 +1830,9 @@ static CURLcode ng_flush_egress(struct Curl_easy *data,
     }
     else {
       timeout = expiry - ts;
+      if(timeout / NGTCP2_MILLISECONDS * NGTCP2_MILLISECONDS < timeout) {
+        timeout += NGTCP2_MILLISECONDS;
+      }
```

Comment:
This logic is a bit funny-looking and I think it could be rephrased to become easier to understand, but if not it could be explained by a comment at least.

Couldn't it instead be written like this? (or maybe I  misunderstand it)
~~~
if(timeout % NGTCP2_MILLISECONDS)
  timeout += NGTCP2_MILLISECONDS;

Reply:
Fix committed via 0593e0259ac27c2351fd1195ff075a6a848d9105